administrator/components/com_admin/sql/updates/postgresql/4.3.0-2022-07-30.sql

```diff
@@ -0,0 +1,96 @@
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS "#__guidedtours"
+(
+  "id" int NOT NULL AUTO_INCREMENT,
+  "asset_id" int DEFAULT 0,
+  "title" varchar(255) NOT NULL,
+  "description" text NOT NULL,
+  "ordering" int NOT NULL DEFAULT 0,
+  "extensions" text NOT NULL,
+  "url" varchar(255) NOT NULL,
+  "overlay" tinyint NOT NULL  DEFAULT 0,
+  "created" datetime NOT NULL,
+  "created_by" int NOT NULL DEFAULT 0,
+  "modified" datetime NOT NULL,
+  "modified_by" int NOT NULL DEFAULT 0,
+  "checked_out_time" datetime NOT NULL,
+  "checked_out" int NOT NULL DEFAULT 0,
+  "published" tinyint NOT NULL DEFAULT 0,
+  "state" tinyint NOT NULL DEFAULT '1',
+  PRIMARY KEY ("id")
+);
+
+CREATE INDEX "#__guidedtours_idx_asset_id" ON "#__guidedtours" ("asset_id");
+CREATE INDEX "#__guidedtours_idx_title" ON "#__guidedtours" ("title"(191));
+CREATE INDEX "#__guidedtours_idx_created" ON "#__guidedtours" ("created");
+CREATE INDEX "#__guidedtours_idx_created_by" ON "#__guidedtours" ("created_by");
+CREATE INDEX "#__guidedtours_idx_modified" ON "#__guidedtours" ("modified");
+CREATE INDEX "#__guidedtours_idx_modified_by" ON "#__guidedtours" ("modified_by");
+CREATE INDEX "#__guidedtours_idx_checked_out" ON "#__guidedtours" ("checked_out");
+
+--
+-- Dumping data for table `#__guidedtours`
+--
+
+INSERT INTO "#__guidedtours" ("id", "asset_id", "title", "description", "ordering", "extensions", "url", "overlay", "created", "created_by", "modified", "modified_by", "checked_out_time", "checked_out", "published") VALUES
+(1, 91, 'How to create a Guided Tour in Joomla Backend?', '<p>This Tour will show you that how you can create Guided Tour in Joomla Backend!</p>', 0, '[\"com_guidedtours\"]', 'administrator/index.php?option=com_guidedtours&view=tours', 1, '', 0, '', 0, '', 0, 1);
```

Comment:
this is not the correction I proposed

Reply:
I have done the required changes. I have made changes in the script files and `extensions.sql` for Mysql and Postgresql.

Reply:
thank you